compiler/noirc_driver/src/lib.rs

```diff
@@ -250,12 +254,14 @@ pub fn check_crate(
     deny_warnings: bool,
     disable_macros: bool,
     use_elaborator: bool,
+    skip_prelude: bool,
```

Comment:
It would probably be good to start grouping these together. It looks like in a lot of tests we just have four `false` variables being passed through.

Maybe we could put these options on the `Context` to easily thread them to other passes as well. This is non-blocking but just a thought